modules/statics/test/unit/coins.ts

```diff
@@ -61,4 +61,78 @@ describe('All Coins', () => {
     dot.decimalPlaces.should.equal(12);
     dot.primaryKeyCurve.should.equal('ed25519');
   });
+
+  describe('ARC20 and AVAXERC20 Tokens', () => {
+    it('Token PNG', () => {
```

Comment:
I recommend the description in the "it" block to be a verb in imperative tense:

`it('should contain definition for Token PNG')`

That way the test case is equivalent to a Business Rule that the software should obey.

Same comment for following tests

Reply:
Done